tensorflow/lite/examples/label_image/label_image.cc

```diff
@@ -101,6 +102,23 @@ TfLiteDelegatePtrMap GetDelegates(Settings* s) {
     }
   }
 
+  if (s->xnnpack_delegate) {
+    TfLiteXNNPackDelegateOptions xnnpack_options =
+        TfLiteXNNPackDelegateOptionsDefault();
+    xnnpack_options.num_threads = s->number_of_threads;
+
+    auto xnnpack_delegate = TfLiteXNNPackDelegateCreate(&xnnpack_options);
+    auto delegate = Interpreter::TfLiteDelegatePtr(
+        xnnpack_delegate, [](TfLiteDelegate* delegate) {
+          TfLiteXNNPackDelegateDelete(delegate);
+        });
```

Comment:
Lets add the above part to `evaluation/utils` like the other delegates?

Reply:
OK, I'll do it.

Reply:
updated